### PR TITLE
Implement new version of `@inubekit foundations`

### PR DESCRIPTION
**Pull request by Andresbl123**
New version of the **@inubekit/foundations** library with new color tokens implemented

### CODE REVIEW

**Review by JanuarAsprilla — APPROVED** [review]

File: src/Text/props.ts [hunk 1]
```diff
@@ -19,11 +19,9 @@ export type HtmlElements =
 
 export type SizeOptions = "large" | "medium" | "small";
```
Comment (wfercanas) [inline src/Text/props.ts:20]: Rename to just Size

File: src/Text/props.ts [hunk 2]
```diff
@@ -19,11 +19,9 @@ export type HtmlElements =
...
 
-export type TypeOptions = keyof typeof inube.typography;
+export type TypeOptions = "body" | "display" | "label" | "title" | "headline";
```
Comment (wfercanas) [inline src/Text/props.ts:22]: Rename to just Type

File: src/Text/props.ts [hunk 3]
```diff
@@ -2,7 +2,7 @@ import { inube } from "@inubekit/foundations";
 
 export type AlignOptions = "start" | "center" | "end" | "justify";
```
Comment (wfercanas) [inline src/Text/props.ts:3]: Rename to just Alignment

File: src/Text/props.ts [hunk 4]
```diff
@@ -2,7 +2,7 @@ import { inube } from "@inubekit/foundations";
...
 
-export type Appearance = keyof typeof inube.color.text;
+export type Appearance = keyof typeof inube.text;
 
 export type HtmlElements =
```
Comment (wfercanas) [inline src/Text/props.ts:7]: Rename to HtmlElement

File: src/Text/props.ts (hunk 3 above)
Comment (wfercanas) [inline src/Text/props.ts:3]: Refactor to create an array for storing the options and then use that array to create the type:
```js
const aligments = ["start", "center", "end", "justify"] as const;
export type Alignment = typeof aligments[number];
```

With this change, you can now avoid repeating the options in the props structure for storybook.

```js
// Don't ❌
textAlign: {
    options: ["start", "center", "end", "justify"],
    control: { type: "select" },
    description: "This prop controls the text-align style property.",
    table: {
      defaultValue: { summary: "start" },
    },
  },

// Do ✅
textAlign: {
    options: alignments,
    control: { type: "select" },
    description: "This prop controls the text-align style property.",
    table: {
      defaultValue: { summary: "start" },
    },
  },
```

**This change applies to all literal types defined in this file. The array should be in plural (alignments, types, sizes, etc), and the type in singular.**

**Review by wfercanas — CHANGES_REQUESTED** [review]

**Review by JanuarAsprilla — APPROVED** [review]

File: src/Text/props.ts [hunk 5]
```diff
@@ -1,29 +1,33 @@
...
+  "p",
+  "span",
+  "legend",
+  "figcaption",
+  "blockquote",
+] as const;
+const sizes = ["large", "medium", "small"] as const;
+const types = ["body", "display", "label", "title", "headline"] as const;
 
-export type Appearance = keyof typeof inube.color.text;
```
Comment (wfercanas) [inline src/Text/props.ts:5]: Create the array `appearances = Object.keys(inube.text)`. With this change:
1. The type can be just keyof typeof appearances 
2. In props, you directly use appearances and not the `Object.keys()` method.

File: src/Text/styles.js [hunk 6]
```diff
@@ -2,40 +2,33 @@ import styled from "styled-components";
...
-  letter-spacing: ${({ $type, $size, theme }) =>
-    theme?.typography?.[$type]?.[$size]?.tracking ||
-    inube.typography[$type][$size].tracking};
-  font-weight: ${({ $type, $size, theme }) =>
-    theme?.typography?.[$type]?.[$size]?.weight ||
-    inube.typography[$type][$size].weight};
+import { typography } from "./typography";
+
+export const StyledText = styled.p`
+  font-family: ${({ theme }) => theme?.font || "Roboto"};
```
Comment (wfercanas) [inline src/Text/styles.js:8]: Extend to include a `sans-serif` as fallback.

**Review by wfercanas — CHANGES_REQUESTED** [review]

**Review by JanuarAsprilla — APPROVED** [review]

File: src/Text/styles.js (hunk 6 above)
Comment (cmarin001) [reply src/Text/styles.js:8]: solved

File: src/Text/props.ts (hunk 5 above)
Comment (cmarin001) [reply src/Text/props.ts:5]: solved

**Review by JanuarAsprilla — APPROVED** [review]

File: src/Text/props.ts (hunk 5 above)
Comment (wfercanas) [reply src/Text/props.ts:5]: Si ya creó el array `appearances`, puede usarlo para crear el type Appearance en lugar de volver a ir a `inube.text`. De esta manera todos los types de este estilo quedan con una sola forma de definición.

**Review by wfercanas — CHANGES_REQUESTED** [review]
Stardardize Appearance type

File: src/Text/props.ts (hunk 5 above)
Comment (cmarin001) [reply src/Text/props.ts:5]: solved

**Review by wfercanas — APPROVED** [review]

**Pull request merged**